AxonDeepSeg/integrity_test.py

```diff
@@ -4,54 +4,59 @@
 # Basic integrity test to check is AxonDeepSeg is correctly installed
 # Launches a segmentation in the data_test folder
 
-try:
+def integrity_test():
 
-    import json
-    import os
-    from AxonDeepSeg.testing.segmentation_scoring import *
-    from time import time
+    try:
 
-    from scipy.misc import imread, imsave
+        import json
+        import os
+        from AxonDeepSeg.testing.segmentation_scoring import *
+        from time import time
+        from AxonDeepSeg.apply_model import axon_segmentation
+        from scipy.misc import imread, imsave
+        dir_path = os.path.dirname(os.path.abspath(__file__))
 
-    # input parameters
+        # input parameters
 
-    path_testing = "../AxonDeepSeg/data_test/"
-    model_name = 'default_SEM_model_v1'
-    path_model = '../AxonDeepSeg/models/' + model_name
+        path_testing = dir_path + "/data_test/"
+        model_name = 'default_SEM_model_v1'
+        path_model = dir_path + '/models/' + model_name
 
+        path_configfile = path_model + '/config_network.json'
 
-    path_configfile = path_model + '/config_network.json'
+        print('Reading test configuration file.')
+        if not os.path.exists(path_model):
+            os.makedirs(path_model)
 
-    if not os.path.exists(path_model):
-        os.makedirs(path_model)
+        with open(path_configfile, 'r') as fd:
+            config_network = json.loads(fd.read())
 
-    with open(path_configfile, 'r') as fd:
-        config_network = json.loads(fd.read())
+        print('Computing the segmentation of axon and myelin on test image.')
+        prediction = axon_segmentation([path_testing], ["image.png"], path_model, config_network,verbosity_level=0)
 
 
-    from AxonDeepSeg.apply_model import axon_segmentation
+        mask = imread(path_testing + '/mask.png', flatten=True)
+        pred = imread(path_testing + '/AxonDeepSeg.png', flatten=True)
 
-    prediction = axon_segmentation([path_testing], ["image.png"], path_model, config_network,verbosity_level=0)
+        print('Generating axon and myelin segmentation masks and saving.')
+        gt_axon = mask > 200
```

Comment:
why this value?

Reply:
To get binary masks of axon-myelin as axons=255, myelin=127.

Reply:
then explain in comments.